docs/custom_quantum_functions.md

```diff
@@ -60,6 +60,8 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
         PLAY_NOTE_ARRAY(tone_qwerty);
       }
       return true; // Let QMK send the enter press/release events
+    default:
```

Comment:
This isn't needed , if it's laid out properly. 

It should be like this: 

```
    return false;
    break;
  }
  return true;
}
```
This is more consistent with how it's set up, and less likely to cause issues, if something is added to the default block.

Reply:
Now that I look at it, it'd have the same effect.
Still, one of the two has to be implemented, it just doesn't work without that ``return true;``.

Reply:
Yeah, it would be. So, it's more of a "consistency" thing, as that's how it is usually presented.

But as long as it compiles, you should be fine.  (And the other examples do show all of this correctly)

Reply:
I think the `default` is sometimes required, so I wouldn't mind that being in the example with this :)